topology/The College of New Jersey/TCNJ - ELSA/ELSA.yaml

```diff
@@ -58,3 +58,30 @@ Resources:
           hidden: false
     Tags:
       - CC*
+  TCNJ-ELSA:
```

Comment:
This looks like a duplicate of TCNJ-ELSA from line 34.

Reply:
The ce was re-deployed with a different more descriptive FQDN.  and the old was set to false.

What is the best way to transition this?

Reply:
Does the old one already have records in GRACC? @djw8605 , is it OK to change the FQDN of an existing resource or would that cause problems?

Reply:
The old one has records in GRACC, and the FQDN was used to match to records.  The old hosted-ce30 needs to stick around for a while.

Reply:
OK. Jeff, you'll need to give the new one a new name (like TCNJ-ELSA2 or something). Use 1137 for the ID.

Reply:
We discussed this in Slack; the existing TCNJ-ELSA entry should be renamed TCNJ-ELSA-old, set inactive, and a new TCNJ-ELSA entry should be created (using the new FQDN, and 1137 for the ID).